playbook/app/pb_kits/playbook/pb_date_time_stacked/_date_time_stacked.scss

```
@@ -1,6 +1,15 @@
-@import "../pb_title/title";
-@import "../pb_body/body";
+[class^=pb_date_time_stacked] {
+
+  [class^=padding_month] {
+    padding-bottom: 10px !important;
```

```suggestion
    padding-bottom: ($space_xs + 2) !important;
```

Let's try and use variables even if they are slightly custom